Description: Upstream changes introduced in version 0.1.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 rpi.gpio (0.1.0-1) unstable; urgency=low
 .
   * Initial release
 .
 The person named in the Author field signed this changelog entry.
Author: Ben Croston <ben@croston.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- rpi.gpio-0.1.0.orig/setup.py
+++ rpi.gpio-0.1.0/setup.py
@@ -1,6 +1,4 @@
 #!/usr/bin/env python
-import distribute_setup
-distribute_setup.use_setuptools()
 from setuptools import setup, find_packages
 import platform
 
